.prettierrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("@khanacademy/eslint-config/.prettierrc.js");
```

Comment:
@jeresig I made the following change after you approved.  This is to ensure that we're using the same prettier config for VSCode as we are for eslint.

Reply:
Oh nice!